pages/index.vue

```diff
@@ -1,59 +1,306 @@
+		<div id="pricing-section" class="active">
+			<div class="container space-2 space-md-3">
+				<div class="w-md-75 w-lg-50 text-center mx-md-auto mb-9">
+					<h2 class="h1 font-weight-normal">
+						خرید بلیط
+					</h2>
+					<p>یک روز کامل با Vue.js و عاشقان کد نویسی</p>
+				</div>
+
+				<div class="row">
+					<link rel="stylesheet" href="">
+					<div id="stickyBlockStartPoint" class="col-md-5 mb-9 mb-md-0">
+						<div
+								class="position-relative shadow-lg text-white rounded z-index-2 p-7"
+								style="background: #35495e;">
+              <span class="d-block">
+                <span class="align-top">تومان</span>
+                <span class="display-4 font-weight-semi-bold">۸۶,۰۰۰</span>
+              </span>
+
+							<hr class="opacity-md my-4">
+
+							<div class="mb-5">
+								<p class="text-light">
+									دانش خود را با ما به اشتراک بگذارید و با بزرگان Vue.Js هم صحبت بشوید.
+								</p>
+							</div>
+
+							<a class="btn btn-white btn-pill transition-3d-hover mb-2" href="#">
+								خرید بلیط
+								<span class="fas fa-arrow-left text-primary font-size-1 mr-2"></span>
+							</a>
+
+							<div class="position-absolute bottom-0 right-0 w-100 max-width-15 z-index-n1">
+								<img src="/svg/index/ticket_bg.svg" alt="Ticket Background">
+							</div>
+						</div>
+					</div>
+
+					<div class="col-md-7">
+						<div class="pl-lg-6">
+							<div class="row">
+								<div class="col-sm-6 mb-5">
+									<h4 class="h5">
+										<i class="fa fa-check-square"></i>
+										قابلیت
+									</h4>
+									<p>
+										لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است.
+									</p>
+								</div>
+								<div class="col-sm-6 mb-5">
+									<h4 class="h5">
+										<i class="fa fa-check-square"></i>
+										ویژگی
+									</h4>
+									<p>
+										لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است.
+									</p>
+								</div>
+							</div>
+
+							<div class="row mb-9">
+								<div class="col-sm-6 mb-5">
+									<h4 class="h5">
+										<i class="fa fa-check-square"></i>
+										فیوچر
+									</h4>
+									<p>
+										لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است.
+									</p>
+								</div>
+								<div class="col-sm-6">
+									<h4 class="h5">
+										<i class="fa fa-check-square"></i>
+										امکانات
+									</h4>
+									<p>
+										لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ و با استفاده از طراحان گرافیک است.
+									</p>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</main>
 </template>
 
 <script>
-import AppIcon from '~/components/AppIcon.vue'
-
-export default {
-  head: {
-    title: 'Homepage'
-  },
-  components: {
-    AppIcon
-  }
-}
+
+	export default {
+		head: {
+			title: 'Homepage'
+		},
+		data() {
+			return {
+				speakers: [
```

Comment:
I would suggest refactoring such static data to somewhere like `~/data/speakers.json`.

Reply:
OK, will surely do it 👌